2-mandatory/1-create-functions.js

```diff
@@ -24,16 +44,30 @@ Write a function that:
 - Removes any forward slashes (/) in the strings.
 - Makes the strings all lowercase.
 */
-function tidyUpString() {
+ 
+function tidyUpString(arr) {
```

Comment:
Good job with this function. Just a minor comment, try to give more meaningful names to the variables and avoid using keywords like 'array' or 'string' as those can get confused with the type.